seed/challenges/03-front-end-libraries/jquery.json

```diff
@@ -1195,12 +1195,13 @@
       "id": "bad87fee1348bd9aed208826",
       "title": "Target the Children of an Element Using jQuery",
       "description": [
-        "Many HTML elements have <code>children</code> which <code>inherit</code> their properties from their parent HTML elements.",
-        "For example, every HTML element is a child of your <code>body</code> element, and your \"jQuery Playground\" <code>h3</code> element is a child of your <code>&#60;div class=\"container-fluid\"&#62</code> element.",
+        "When HTML elements are placed one level below another they are called <code>children</code> of that element. For example, the button elements in this challenge with the text \"#target1\", \"#target2\", and \"#target3\" are all <code>children</code> of the <code>&#60;div class=\"well\" id=\"left-well\"&#62;</code> element.",
         "jQuery has a function called <code>children()</code> that allows you to access the children of whichever element you've selected.",
-        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color of blue:",
+        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color blue:",
         "<code>$(\"#left-well\").children().css(\"color\", \"blue\")</code>",
-        "Give all the children of your <code>#right-well</code> element a color of orange."
+        "<hr>",
+        "Give all the children of your <code>right-well</code> element the color orange."
+
```

Comment:
@Ntropish Yes, that is correct. The diff shows that this JSON has an extra line. If you are stuck, you can let us know in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), we will try and help.